cmd/kpod/stop.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	defaultTimeout int64 = 10
+	stopFlags            = []cli.Flag{
+		cli.Int64Flag{
+			Name:  "timeout, t",
+			Usage: "Seconds to wait for stop before killing the container (default 10)",
+			Value: defaultTimeout,
+		},
+	}
+	stopDescription = "Stop one or more containers"
+	stopCommand     = cli.Command{
+		Name: "stop",
+		Usage: "Stops one ore more running containers.  The container name or ID can be used.  A timeout to forcibly" +
+			"stop the container can also be set but defaults to 10 seconds otherwise.",
```

Comment:
Shouldn't this start with a space? We're concatenating it to the last string, so it will read "forciblystop" now

Reply:
fixed